testing/trino-tests/src/test/java/io/trino/security/TestingSystemSecurityMetadata.java

```diff
@@ -259,4 +259,13 @@ public void tableRenamed(Session session, CatalogSchemaTableName sourceTable, Ca
 
     @Override
     public void tableDropped(Session session, CatalogSchemaTableName table) {}
+
+    @Override
+    public void columnCreated(Session session, CatalogSchemaTableName table, String column) {}
+
+    @Override
+    public void columnRenamed(Session session, CatalogSchemaTableName table, String oldName, String newName) {}
+
+    @Override
+    public void columnDropped(Session session, CatalogSchemaTableName table, String column) {}
```

Comment:
How about throwing `UnsupportedOperationException`?

Reply:
I can add; was just trying to maintain consistency with schema/table events. Those are all no-ops as well.